lib/easy.c

```diff
@@ -567,6 +567,41 @@ static unsigned int populate_fds(struct pollfd *fds, struct events *ev)
   return numfds;
 }
 
+/* poll_fds()
+ *
+ * poll the fds[] array
+ */
+static int poll_fds(struct events *ev,
+                    struct pollfd *fds,
+                    const unsigned int numfds,
+                    CURLcode *result)
```

Comment:
I think you should rather make it return `CURLcode` which is the common pattern in the code, and then thus return pollrc in an argument. Switch the arguments.